### PR TITLE
MAT-3711 add patient end point and insert to db

### PR DESCRIPTION
**Pull request by sb-cecilialiu**

### CODE REVIEW

File: src/main/java/com/example/template/models/Patient.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import org.springframework.data.annotation.Id;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Patient {
+  @Id private String id;
```
Comment (adongare) [inline src/main/java/com/example/template/models/Patient.java:13]: I think mongo `ObjectId` is better than string.

File: src/main/java/com/example/template/controller/PatientController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.template.models.Patient;
+import com.example.template.repositories.PatientRepository;
```
Comment (adongare) [inline src/main/java/com/example/template/controller/PatientController.java:13]: probably package structure needs to be updated.

Comment (sb-cecilialiu) [reply src/main/java/com/example/template/controller/PatientController.java:13]: @adongare Thanks Ashok! Can you please let me know what package structure should it be?

File: src/main/java/com/example/template/models/Patient.java (hunk 1 above)
Comment (adongare) [reply src/main/java/com/example/template/models/Patient.java:13]: Pls ignore my comment. I think @gregory-akins changed it to String for some reason in measure model(measure service). You can leave it as string here as well unless someone has any objection.

Comment (sb-cecilialiu) [reply src/main/java/com/example/template/models/Patient.java:13]: @adongare Ok thanks so much for letting me know! Appreciate it!!

**Review by gregory-akins — APPROVED** [review]

**Pull request merged**